nilearn/decomposition/tests/test_decomposition_estimators.py

```diff
@@ -23,7 +23,6 @@
 )
 
 ESTIMATORS_TO_CHECK = [
-    _MultiPCA(verbose=0),
```

Comment:
do not test private class as it should already be tested by classes that inherit from it